ethereum/executionlayer/src/test/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionLayerBlockProductionManagerImplTest.java

```diff
@@ -111,6 +114,9 @@ public void preDeneb_builderOffline() throws Exception {
         .isEqualTo(executionPayloadContext);
     assertThat(executionPayloadResult.getExecutionPayloadFuture()).isEmpty();
     assertThat(executionPayloadResult.getBlobsBundleFuture()).isEmpty();
+    assertThat(executionPayloadResult.getExecutionPayloadValueFuture()).isPresent();
```

Comment:
I'd add `verify(executionClientHandler).engineGetPayload(any(), any())` here (which is good idea to clarify when it's fixed), and I'm getting error, because it was 2 times. The reason is we are calling it 2 times (see last comment to `ExecutionLayerBlockProductionManagerImpl`) which is not correct I think.

Reply:
Done